src/platform/silabs/wifi/DnssdImpl.cpp

```diff
@@ -0,0 +1,172 @@
+#include "lib/dnssd/platform/Dnssd.h"
+#include "platform/CHIPDeviceLayer.h"
+extern "C" {
+#include "sl_mdns.h"
+}
+#include <lib/support/CHIPMem.h>
+#include <lib/support/CodeUtils.h>
+#include <lib/support/logging/CHIPLogging.h>
+
+using namespace ::chip::DeviceLayer;
+
+namespace chip {
+namespace Dnssd {
+
+// Global mDNS instance
+static sl_mdns_t gMdnsInstance;
```

Comment:
Be definition static variable are not global. Also, this should in an anounymous namespace and not static.

Reply:
moved